src/dragonball/src/metric.rs

```diff
@@ -55,4 +63,12 @@ pub struct DragonballMetrics {
     pub seccomp: SeccompMetrics,
     /// Metrics related to signals.
     pub signals: SignalMetrics,
+    /// Metrics related to i8032 device.
+    #[cfg(target_arch = "x86_64")]
+    pub i8042: HashMap<String, Arc<I8042DeviceMetrics>>,
```

Comment:
we only have one i8042 device and one rtc device, so I think we don't need a hashmap here.